CEDS/MSSQL/0037-View-FactK12ProgramParticipationsDim-Create.sql

```diff
@@ -0,0 +1,213 @@
+				ON StudentEducationOrganizationAssociation.StudentUSI = HomelessUnaccompaniedYouthCharacteristic.StudentUSI
+		LEFT JOIN
+			analytics_config.ceds_TableInformation HomelessUnaccompaniedYouth
+				ON HomelessUnaccompaniedYouthCharacteristic.StudentCharacteristicDescriptorId = HomelessUnaccompaniedYouth.DescriptorId
+					AND ceds_K12DemographicDim.HomelessUnaccompaniedYouthStatusCode = HomelessUnaccompaniedYouth.CodeValue
+		LEFT JOIN
+			analytics_config.ceds_TableInformation SexCode
+				ON StudentEducationOrganizationAssociation.SexDescriptorId = SexCode.DescriptorId
+					AND ceds_K12DemographicDim.SexCode = SexCode.CodeValue
+        LEFT JOIN
+            edfi.StudentHomelessProgramAssociation
+                ON StudentHomelessProgramAssociation.StudentUSI = StudentEducationOrganizationAssociation.StudentUSI
+        INNER JOIN
+            edfi.GeneralStudentProgramAssociation
+                ON
+                    GeneralStudentProgramAssociation.BeginDate = StudentSpecialEducationProgramAssociation.BeginDate
+                    OR GeneralStudentProgramAssociation.EducationOrganizationId = StudentSpecialEducationProgramAssociation.EducationOrganizationId
+                    OR GeneralStudentProgramAssociation.ProgramEducationOrganizationId = StudentSpecialEducationProgramAssociation.ProgramEducationOrganizationId
+                    OR GeneralStudentProgramAssociation.ProgramName = StudentSpecialEducationProgramAssociation.ProgramName
+                    OR GeneralStudentProgramAssociation.ProgramTypeDescriptorId = StudentSpecialEducationProgramAssociation.ProgramTypeDescriptorId
+                    OR GeneralStudentProgramAssociation.StudentUSI = StudentSpecialEducationProgramAssociation.StudentUSI
+        INNER JOIN 
+            edfi.StudentSpecialEducationProgramAssociationDisability
+                ON
+                    StudentSpecialEducationProgramAssociationDisability.BeginDate = StudentSpecialEducationProgramAssociation.BeginDate
+                    OR StudentSpecialEducationProgramAssociationDisability.EducationOrganizationId = StudentSpecialEducationProgramAssociation.EducationOrganizationId
+                    OR StudentSpecialEducationProgramAssociationDisability.ProgramEducationOrganizationId = StudentSpecialEducationProgramAssociation.ProgramEducationOrganizationId
+                    OR StudentSpecialEducationProgramAssociationDisability.ProgramTypeDescriptorId = StudentSpecialEducationProgramAssociation.ProgramTypeDescriptorId
+                    OR StudentSpecialEducationProgramAssociationDisability.StudentUSI = StudentSpecialEducationProgramAssociation.StudentUSI
+        INNER JOIN 
+            analytics.ceds_IdeaStatusDim DimIdeaStatus
+                ON
+                    Descriptor.CodeValue = DimIdeaStatus.BasisOfExitCode 
+                    AND Descriptor.CodeValue = DimIdeaStatus.DisabilityCode
```

Comment:
I think you are mixing different descriptor here as well., because this descriptor is the one you used for `ProgramType`.